utils/go.mod

```diff
@@ -93,6 +93,7 @@ require (
 	github.com/tklauser/numcpus v0.6.1 // indirect
 	github.com/xanzy/ssh-agent v0.3.3 // indirect
 	github.com/yusufpapurcu/wmi v1.2.4 // indirect
+	go.uber.org/mock v0.4.0 // indirect
```

Comment:
I don't think this dependency is necessary. I believe it may have been brought up when you regenerate the mocks. Could you keep the golang standard library implementation?